src/ducks/WatchlistAddressesTable/index.js

```diff
@@ -50,7 +50,11 @@ const normalizeCSVData = items => items.map(normalizeCSVItem)
 const WatchlistAddressesTable = props => {
   const [list, setList] = useState(props.watchlist)
   const { activeColumns, setActiveColumnsKeys } = useColumns(BLOCKCHAIN_ADDRESS)
-  const columns = useMemo(() => combineColumns(activeColumns), [activeColumns])
+  const columns = useMemo(() => {
+    const _columns = combineColumns(activeColumns)
+    if (props.isDesktop) return _columns
+    return _columns.filter(({ id }) => id !== 'CHECKBOX')
```

Comment:
1. `_columns` can be renamed to `columns` since it's a new block scope;
2. Better to use the id of the column object `CHECKBOX_COLUMN` instead of arbitrary string